src/components/study-manager.js

```diff
@@ -384,18 +396,24 @@ const StudyCard = ({ study, onClick, studyCreationLoader }) => {
                 </CardActions>
                 <Collapse in={expanded} timeout="auto" unmountOnExit>
                     <CardContent>
-                        <Typography variant="button">
-                            <FormattedMessage id="studyName" />
-                        </Typography>
-                        <Typography variant="body2" paragraph>
-                            {study.studyName}
-                        </Typography>
-                        <Typography variant="button">
-                            <FormattedMessage id="studyDescription" />
-                        </Typography>
-                        <Typography variant="body2" paragraph>
-                            {study.description ? study.description : '—'}
-                        </Typography>
+                        <CustomTypography className={classes.label}>
```

Comment:
What is the purpose of classes.label which does not exist and using a custom styled component?

Reply:
I agree, why this change to customTypography ?